### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by apaleslimghost**
Needs design review. Do the new states and colours make sense? This is technically semver-major since the semantics and format of `$o-labels-states` have changed, but it was never documented and I'd be willing to bet nobody used it.

New hotness:

screenshot 2016-10-07 17 34 16
screenshot 2016-10-07 17 34 08

Old and busted:

screenshot 2016-10-07 17 35 01

### CODE REVIEW

File: scss/_states.scss [hunk 1]
```diff
@@ -4,28 +4,47 @@
...
+	),
+	brand: (
+		text: 'white',
+		background: 'claret'
 	)
 ) !default;
+
+@each $state, $colors in $o-labels-states {
+	@each $usecase, $palette in $colors {
+		@include oColorsSetUseCase(o-labels-#{$state}, $usecase, $palette);
```
Comment (JakeChampion) [inline scss/_states.scss:48]: Real nice! 👍

**Review by JakeChampion — APPROVED** [review]

File: scss/_mixins.scss [hunk 2]
```diff
@@ -13,6 +16,11 @@
 ///
 /// @param {String} $size
 @mixin oLabelsSize($size) {
+	@if not $_o-labels-warned-size and ($size == 'medium' or $size == 'big') {
```
Comment (onishiweb) [inline scss/_mixins.scss:19]: Was this a design decision to use `o-typography` and remove the label sizing?

**Review by onishiweb — CHANGES_REQUESTED** [review]

**Comment by onishiweb** [comment]
I'm not sure about it being a major release? Is there anything in it that will break existing labels apart from the update their styling?

Also, has this been through design review yet?

Thanks for the PR :)

**Comment by apaleslimghost** [comment]
If anyone were using `$o-labels-states` this would be breaking, but I'm pretty sure I'm the only user of this module, and I don't use it.

I'm going to find a designer to review it this afternoon.

**Comment by onishiweb** [comment]
Cool, thanks. Ok, in that case it's breaking, just in case someone other than you is using it.

**Comment by apaleslimghost** [comment]
Sue's happy with the new design, agrees with the o-typography sizing @onishiweb

**Review by onishiweb — APPROVED** [review]

**Comment by chee** [comment]
yes i think the colours make sense

**Pull request merged**